### PR TITLE
Enhancements/general

### PR DESCRIPTION
**Pull request by boboeyen**
Some little adjustments to enhance the usability
- Improvement of a definition in radiopropa raytracer
- including loggers and logger levels in some modules
- including a standard uniform ice model

### CODE REVIEW

File: NuRadioMC/utilities/medium.py [hunk 1]
```diff
@@ -276,7 +276,17 @@ def get_ice_model_radiopropa(self,discontinuity=False):
         return ice
 
 
-
+class uniform_ice(medium_base.IceModelSimple):
+    """
+    unifrom ice with refractive index of typical deep ice (1.78)
```
Comment (anelles) [inline NuRadioMC/utilities/medium.py:281]: spelling: uniform

**Review by anelles — COMMENTED** [review]
Looks good. Tiny spelling improvement.

File: NuRadioMC/utilities/medium.py (hunk 1 above)
Comment (boboeyen) [reply NuRadioMC/utilities/medium.py]: oops

**Review by anelles — APPROVED** [review]
All changes look good to me.

**Review by cg-laser — COMMENTED** [review]
looks good but please try to wait at least 24h before merging after a positive review. I didn't have the chance to look at the changes before you merged it.

**Comment by boboeyen** [comment]
Oh right, forgot about that, sorry!

**Pull request merged**